org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandler.java

```diff
@@ -58,32 +63,38 @@ public InitHandler(ProjectsManager manager, PreferenceManager preferenceManager,
 		this.preferenceManager = preferenceManager;
 	}
 
-
-	InitializeResult initialize(InitializeParams param){
-		logInfo("Initializing Java Language Server "+JavaLanguageServerPlugin.getVersion());
-		if(param.getCapabilities()==null){
+	InitializeResult initialize(InitializeParams param) {
+		logInfo("Initializing Java Language Server " + JavaLanguageServerPlugin.getVersion());
+		if (param.getCapabilities() == null) {
 			preferenceManager.updateClientPrefences(new ClientCapabilities());
-		}else{
+		} else {
 			preferenceManager.updateClientPrefences(param.getCapabilities());
 		}
 		String rootPath = param.getRootUri();
 		if (rootPath != null) {
 			URI uri = URI.create(rootPath);
-			if ("file".equals(uri.getScheme())){
+			if ("file".equals(uri.getScheme())) {
```

Comment:
I'm not quite sure what the logic here is: if we only accept file URI's, but get a "http" uri, should we not fail? we're not guarding against that.

Reply:
Some unintended format change. Will revert those